example/styles.scss

```diff
@@ -1 +1,3 @@
-/* You can add global styles to this file, and also import other style files */
\ No newline at end of file
+/* You can add global styles to this file, and also import other style files */
+$icon-font-path: '~bootstrap-sass/assets/fonts/bootstrap/';
```

Comment:
Unfortunately not, but this was giving `GET` request error while fetching font files before, now there is no error but they aren't displayed :(